style.css

```diff
@@ -1 +1,156 @@
-/*  Add your styling here */
+/*  DEFAULT */
+:root {
+  --main-color: #faf1da;
+  --font-color: #5f4501;
+  --third-color: #9c6644;
+}
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+li {
+  list-style: none;
+}
+
+body {
+  font-family: "Lato", sans-serif;
+}
+
+header,
+footer {
+  background-color: var(--main-color);
+  padding: 1rem;
+  color: var(--font-color);
+}
+
+.banner {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 90px;
+  height: 90px;
+}
+
+.header-shop-name {
+  font-family: "Lobster", cursive;
+  font-size: 1.3rem;
+  padding: 0.8rem;
+}
+
+h3.header-text {
+  width: 250px;
+  position: relative;
+  left: 8vw;
+  font-size: 1rem;
+  bottom: 10%;
+  padding: 2rem 1rem 1rem 1rem;
```

Comment:
You could also write `2rem 1rem 1rem` here (since left and right padding are the same)